### ISSUE TITLE
JPA Diagram from Oracle - Slow Available Tables fetch

### ISSUE THREAD

**Issue by joefor** [jGauravGupta/JPAModeler#63]
When connecting to an Oracle 11 database and selecting a schema with 123 tables, the tool is very slow (~4-6 minutes) in processing the tables before the "Available Tables" section is available. In my use case, I only care about 4 of them for a particular application I am working on.

Is it pre-fetching all the table meta data up front for all tables in the selected schema? Is there a way to disable this?

**Comment by jGauravGupta** [comment]
Yes, It is pre-fetching all the table meta data to find all the related table( entity relationship).
**Reason :** If u look at **"Related Table"** checkbox in the same panel, It helps to detect the connected table and includes the related table in **Selected Table** category automatically.

This panel is part of NetBeans API so will check the possible way to disable it.
